modules/loader-utils/src/lib/request-utils/request-scheduler.ts

```diff
@@ -2,7 +2,7 @@ import {Stats} from '@probe.gl/stats';
 
 type Handle = any;
 type DoneFunction = () => any;
-type GetPriorityFunction = () => number;
+type GetPriorityFunction<T = Handle> = (handle: T) => number;
```

Comment:
Should this be `handle?:`? It doesn't always require an argument, right?

Reply:
It should be ok.
I've aligned typings with implementation and i assumed it's ok. See `_updateRequest`, where `getPriority` is called with `handle`.